source/adapters/level_zero/adapter.cpp

```diff
@@ -184,7 +184,7 @@ UR_APIEXPORT ur_result_t UR_APICALL urAdapterGetLastError(
   std::ignore = Adapter;
   std::ignore = Message;
   std::ignore = Error;
-  urPrint("[UR][L0] %s function not implemented!\n", __FUNCTION__);
+  logger::error("[UR][L0] {} function not implemented!", __FUNCTION__);
```

Comment:
As the name of the logger is `level_zero`, the beginning of this logger output will look like that, I think: `<level_zero>[ERROR][UR][L0]`. No need for double level zero info, I suppose.

Reply:
Done